<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbwindow</class>
 <widget class="QWidget" name="dbwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="queryEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>381</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="databaseEdit">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
    color: #ffffff;               /* –¶–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ */
    background-color: #000000;    /* –¶–≤–µ—Ç —Ñ–æ–Ω–∞ */
    border: 2px solid #555555;    /* –ì—Ä–∞–Ω–∏—Ü–∞ */
    border-radius: 15px;          /* –ó–∞–∫—Ä—É–≥–ª–µ–Ω–∏–µ —É–≥–ª–æ–≤ */
    padding: 5px;                 /* –û—Ç—Å—Ç—É–ø –≤–Ω—É—Ç—Ä–∏ */
    font: 900 12pt &quot;Segoe UI&quot;;    /* –®—Ä–∏—Ñ—Ç (–∂–∏—Ä–Ω—ã–π, —Ä–∞–∑–º–µ—Ä, —Å–µ–º–µ–π—Å—Ç–≤–æ) */
    min-width: 180px;             /* –ú–∏–Ω–∏–º–∞–ª—å–Ω–∞—è —à–∏—Ä–∏–Ω–∞ */
    min-height: 35px;             /* –ú–∏–Ω–∏–º–∞–ª—å–Ω–∞—è –≤—ã—Å–æ—Ç–∞ */
    selection-color: #ff0000;     /* –¶–≤–µ—Ç –≤—ã–¥–µ–ª–µ–Ω–Ω–æ–≥–æ —Ç–µ–∫—Å—Ç–∞ */
    selection-background-color: #00ff00; /* –§–æ–Ω –≤—ã–¥–µ–ª–µ–Ω–Ω–æ–≥–æ —Ç–µ–∫—Å—Ç–∞ */
}

/* –°—Ç–∏–ª—å –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */
QTextEdit:hover {
    border: 2px solid #888888;
}

/* –°—Ç–∏–ª—å –ø—Ä–∏ —Ñ–æ–∫—É—Å–µ */
QTextEdit:focus {
    border: 2px solid #0078d7;
}

/* –°—Ç–∏–ª—å –¥–ª—è –æ—Ç–∫–ª—é—á–µ–Ω–Ω–æ–≥–æ —Å–æ—Å—Ç–æ—è–Ω–∏—è */
QTextEdit:disabled {
    color: #aaaaaa;
    background-color: #333333;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ü†î</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_do">
      <property name="text">
       <string>–≤—ã–ø–æ–ª–Ω–∏—Ç—å</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backup">
      <property name="text">
       <string>—Å–∫–∞—á–∞—Ç—å</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_on">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>–ø–æ–¥–∫–ª—é—á–∏—Ç—å</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>660</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Frame 25 (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>tableView</zorder>
  <zorder>queryEdit</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>backButton</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>pushButton_on</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
